lib/palindrome_permutation.rb

```diff
@@ -1,5 +1,18 @@
 
 def palindrome_permutation?(string)
+  hash = {}
 
- 
-end # method
+  string.length.times do |index|
+    letter = string[index]
+    if hash.keys.include? letter
+      hash[letter] += 1
+    else
+      hash[letter] = 1
+    end
+  end
+
+  ones = hash.values.count(1)
+  twos = hash.values.count(2)
```

Comment:
A letter could appear more than 2 times, for example:  "oooooo" is a palindrome.

You could even have:  "ololo" which is also a palindrome.  You really want to count the number of letters that appear an odd number of times.